content/create-release.md

```diff
@@ -864,6 +864,13 @@ Upload the new dev release.
 
 Assuming you are in the release directory, no path is needed with the above command.
 
+Upload stem cells:
+
+If your BOSH director does not have any stem cells yet, you can find the latest versions [here](https://bosh.cloudfoundry.org/stemcells/).  Select the cloud provider you are using to find the OS version you want to use.  The command to send to BOSH will look like this:
```

Comment:
Could we switch this to a relative link to the stemcells.md file? We also use https://bosh.io/stemcells/ more consistently than bosh.cloudfoundry.org. It may be enough to link to the documentation here: https://bosh.io/docs/uploading-stemcells/#upload